presto-accumulo-iterators/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>334-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-accumulo-iterators</artifactId>
+    <description>Accumulo Iterators for the Presto - Accumulo Connector</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.accumulo.version>1.7.4</dep.accumulo.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-core</artifactId>
+            <version>${dep.accumulo.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
I think we could make this
```
<exclusions>
    <exclusion>
        <groupId>*</groupId>
        <artifactId>*</artifactId>
    </exclusion>
</exclusions>
```
Since we don't need any dependencies, just a few APIs from the core JAR.

Also, make this `<scope>provided</scope>`, as this library will be loaded into Accumulo and thus the Accumulo classes will be provided at runtime.